### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by amaredeus**
## 2.8.0

1. Remove poap delivery via dm, store poap for user
2. Add platform to `/claim`
3. Rename isDMEnabled flag -> isWalletDeliveryEnabled
   - add ethWalletSettings to discordUsers
   - disable DM delivery
4. Update poap expiration only for dates longer than 24 hours
5. Fix duplicate poap assignment
6. Enhance poap claim flow

### CODE REVIEW

File: src/app/commands/poap/Claim.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {
-	CommandContext,
+	CommandContext, CommandOptionType,
```
Comment (nonsensecodes) [inline src/app/commands/poap/Claim.ts:2]: Will you put this on its own line for consistency?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Awesome. Just one change requested for this PR.

File: src/app/commands/poap/Claim.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/commands/poap/Claim.ts:2]: yeah idk why the IDE keeps placing it on the same line

**Review by nonsensecodes — APPROVED** [review]
Perfect. Thank you, @amaredeus!

**Comment by amaredeus** [comment]
Mering as this is an important stability update for the new bot

**Pull request merged**